### PR TITLE
fix(esl-share): make esl:share:ready event bubbled

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

**Review by abarmina — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

File: src/modules/esl-share/core/esl-share-button.ts [hunk 1]
```diff
@@ -199,7 +199,7 @@ export class ESLShareButton extends ESLBaseElement {
   private onReady(): void {
     if (this.ready) return;
     this.$$attr('ready', true);
-    this.$$fire(this.SHARE_READY_EVENT, {bubbles: false});
+    this.$$fire(this.SHARE_READY_EVENT, {bubbles: true});
```
Comment (ala-n) [inline src/modules/esl-share/core/esl-share-button.ts:202]:
```suggestion
    this.$$fire(this.SHARE_READY_EVENT);
```

File: src/modules/esl-share/core/esl-share.ts [hunk 2]
```diff
@@ -118,7 +118,7 @@ export class ESLShare extends ESLTrigger {
   private onReady(): void {
     if (this.ready) return;
     this.$$attr('ready', true);
-    this.$$fire(this.SHARE_READY_EVENT, {bubbles: false});
+    this.$$fire(this.SHARE_READY_EVENT, {bubbles: true});
```
Comment (ala-n) [inline src/modules/esl-share/core/esl-share.ts:121]:
```suggestion
    this.$$fire(this.SHARE_READY_EVENT);
```

File: src/modules/esl-share/core/esl-share-list.ts [hunk 3]
```diff
@@ -86,7 +86,7 @@ export class ESLShareList extends ESLBaseElement {
   private onReady(): void {
     if (this.ready) return;
     this.$$attr('ready', true);
-    this.$$fire(this.SHARE_READY_EVENT, {bubbles: false});
+    this.$$fire(this.SHARE_READY_EVENT, {bubbles: true});
```
Comment (ala-n) [inline src/modules/esl-share/core/esl-share-list.ts:89]:
```suggestion
    this.$$fire(this.SHARE_READY_EVENT);
```

**Review by ala-n — COMMENTED** [review]
Also in case we have it bubbled I'd recommend to prevent or eliminate events of the `esl-share-button` in case the component is inside of container (`esl-share-list`)

**Review by NastaLeo — APPROVED** [review]

**Comment by ala-n** [comment]
Closed, to be discussed if any issue appears in future

**Pull request closed without merging**